src/lib/edi.js

```diff
@@ -216,28 +216,45 @@ class EdiSegmentGroup extends EdiBase {
   }
 
   parse(reader) {
-    var current, segs;
-
+    var current, segs, nest_order;
+    
     //continue in the captcha group until no more matches)
-    while(reader.current() && reader.current().tag === this.config.groupTag) {
+    // allow for explicit nesting numbers
+    while (reader.current() && _.startsWith(reader.current().tag, this.config.groupTag))  {
```

Comment:
do you have an example of what the TAG and GROUP would look like here?

Reply:
Ok think I found it? `TXT:14+Line2'` where the ":" is the separator and hence why starts with.

```
tagRegEx = new RegExp(this.config.groupTag + "(*.)");
while (reader.current() && match = tagRegEx.exec(reader.current().tag)) {
   nest_order = match[1];
   segs = segs || [];
   ...
```